modules/nodes-pods-autoscaling-custom-creating-workload.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-creating-workload_{context}"]
+= Adding a custom metrics autoscaler to a workload
```

Comment:
@zroubalik @weinliu Have we tested all of the parameters and behaviors described in the KEDA docs? I don't feel comfortable documenting behaviors we have not tested. 

For example, this seems like important information. But, do we know this to be true and works as stated?

> KEDA will keep track of the number of consecutive times each scaler has failed to get metrics from its source. Once that value passes the failureThreshold, instead of not propagating a metric to the HPA (the default error behaviour), the scaler will, instead, return a normalised metric using the formula:
> 
> target metric value * fallback replicas
> 
> Due to the HPA metric being of type AverageValue (see below), this will have the effect of the HPA scaling the deployment to the defined number of fallback replicas.

There are similar behaviors throughout their docs.

Reply:
I added a link to the KEDA docs for this behavior and others. Even that I am a bit uncomfortable with.